website/docs/concurrency_control.md

```diff
@@ -2,105 +2,126 @@
+
+#### HiveMetastore based lock provider
 
 ```
 hoodie.write.lock.provider=org.apache.hudi.hive.transaction.lock.HiveMetastoreBasedLockProvider
-hoodie.write.lock.hivemetastore.database
-hoodie.write.lock.hivemetastore.table
 ```
+Following are the basic configs required to setup this lock provider:
 
-`The HiveMetastore URI's are picked up from the hadoop configuration file loaded during runtime.`
-
-**`Amazon DynamoDB`** based lock provider
+| Config Name| Default| Description                                                                                                                                                                                                |
+| ----------------------------------------------------------------------- | ------------------------ |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| hoodie.write.lock.hivemetastore.database                     | N/A **(Required)**                                                                                 | For Hive based lock provider, the Hive database to acquire lock against<br /><br />`Config Param: HIVE_DATABASE_NAME`<br />`Since Version: 0.8.0`                                                                                                                                    |
+| hoodie.write.lock.hivemetastore.table                         | N/A **(Required)**                                                                                 | For Hive based lock provider, the Hive table to acquire lock against<br /><br />`Config Param: HIVE_TABLE_NAME`<br />`Since Version: 0.8.0`                                                                                                                                          |
 
-Amazon DynamoDB based lock provides a simple way to support multi writing across different clusters.  You can refer to the
-[DynamoDB based Locks Configurations](https://hudi.apache.org/docs/configurations#DynamoDB-based-Locks-Configurations)
-section for the details of each related configuration knob.
+`The HiveMetastore URI's are picked up from the hadoop configuration file loaded during runtime.`
 
+#### Amazon DynamoDB based lock provider
 ```
 hoodie.write.lock.provider=org.apache.hudi.aws.transaction.lock.DynamoDBBasedLockProvider
-hoodie.write.lock.dynamodb.table (required)
-hoodie.write.lock.dynamodb.partition_key (optional)
-hoodie.write.lock.dynamodb.region (optional)
-hoodie.write.lock.dynamodb.endpoint_url (optional)
-hoodie.write.lock.dynamodb.billing_mode (optional)
 ```
+Amazon DynamoDB based lock provides a simple way to support multi writing across different clusters.  You can refer to the
```

Comment:
why we are not adding all configs from the list. 
for eg, prior to this patch we listed 

hoodie.write.lock.dynamodb.table (required)
hoodie.write.lock.dynamodb.partition_key (optional)
hoodie.write.lock.dynamodb.region (optional)
hoodie.write.lock.dynamodb.endpoint_url (optional)
hoodie.write.lock.dynamodb.billing_mode (optional)

with the new table, we only have endpoint url.

Reply:
oh, you want to keep this succinct is it. gotcha.